packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/Primes.js

```diff
@@ -29,6 +29,7 @@ function Primes({ name, onChange }) {
                       <NumberInput
                         {...input}
                         size="10"
+                        type="number"
```

Comment:
I would revert these change and later use do a PR to replace that with the <CurrencyFiled/>

Reply:
The `CurrencyWrapper` ? Yep I can do it on this PR !

Reply:
no I prefer do it in another PR ;)